### PR TITLE
feat: update ui with tailwind

### PR DESCRIPTION
**Pull request by zawan-ila**
### Summary

This PR updates the UI to look nicer.

### Instructions

Same as the ones [here](https://github.com/zawan-ila/arbicoin-frontend/pull/5). Just use `tailwind-ui`
 in place of `userflow` in branch name
### Pictures

Some pictures of our new UI are as follows :-

Screenshot 2022-09-13 at 9 25 51 AM
Screenshot 2022-09-13 at 9 26 20 AM
Screenshot 2022-09-13 at 9 26 27 AM
Screenshot 2022-09-13 at 9 26 34 AM
Screenshot 2022-09-13 at 9 26 41 AM
Screenshot 2022-09-13 at 9 27 29 AM

### CODE REVIEW

File: src/components/Blocks/BlockDetail.jsx [hunk 1]
```diff
@@ -3,44 +3,46 @@ import { useParams, Link } from 'react-router-dom'
 
 import { UserContext } from '../User/handleUser'
 
-import '../../common.css'
+// import '../../common.css'
```
Comment (azanbinzahid) [inline src/components/Blocks/BlockDetail.jsx]: please remove this and delete the file if not needed

File: src/components/Blocks/BlockDetail.jsx [hunk 2]
```diff
@@ -3,44 +3,46 @@ import { useParams, Link } from 'react-router-dom'
...
 export default function BlockDetail () {
   const { height } = useParams()
   const [blockAttributes, setBlockAttributes] = useState(null)
   const { myAxios } = useContext(UserContext)
   useEffect(() => {
-    myAxios.get(process.env.REACT_APP_BACKEND_URL + `blocks/height/${height}`).then(res => {
+    myAxios.get(process.env.REACT_APP_BACKEND_URL + `blocks/height/${height}/`).then(res => {
       setBlockAttributes(prevInfo => res.data)
-    }).catch(err => console.log(err))
+    }).catch(err => console.log(err, 123))
```
Comment (azanbinzahid) [inline src/components/Blocks/BlockDetail.jsx]: remove this logging or add meaningful errors.

File: src/components/User/Signup.js [hunk 3]
```diff
@@ -1,23 +1,67 @@
...
     const u = e.target.username.value
     const p = e.target.password.value
-    handleSignup(u, p)
+    const res = await handleSignup(u, p)
+
+    if (!res) {
+      setFormError(true)
+    } else {
+      setFormError(false)
+    }
```
Comment (azanbinzahid) [inline src/components/User/Signup.js]: nit: `condition ? true_expression : false_expression`

**Review by azanbinzahid — COMMENTED** [review]
I see a lot of complex (and long list of) css classes, can we clean them somehow? Also, make sure there's a symmetry followed in the deisgn like same len/width of boxes, left/right aligned texts, consistent color scheme etc.

File: src/components/Blocks/BlockDetail.jsx (hunk 1 above)
Comment (zawan-ila) [reply src/components/Blocks/BlockDetail.jsx]: done

File: src/components/Blocks/BlockDetail.jsx (hunk 2 above)
Comment (zawan-ila) [reply src/components/Blocks/BlockDetail.jsx]: done

File: src/components/User/Signup.js (hunk 3 above)
Comment (zawan-ila) [reply src/components/User/Signup.js]: down

**Comment by zawan-ila** [comment]
> I see a lot of complex (and long list of) css classes, can we clean them somehow? Also, make sure there's a symmetry followed in the deisgn like same len/width of boxes, left/right aligned texts, consistent color scheme etc.

I have tried to make. the css more nicer by making classes to capture common styles....however 'long complex list of css classes' is a popular criticism of tailwind : - (

Personally I feel that the benefits of having your css and styles in one place (as tailwind does) are greater than any 'issues' due to long lists of classes (so long as they aren't too long. I have seen tailwind code with 2-4 lines of css classes : - ) )

**Review by azanbinzahid — APPROVED** [review]

**Pull request merged**